recipes/sliceable-dict/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "sliceable-dict" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 3a47bdd8376bf51d9bf9a21fa08446c376c0b130f2fb8629031e974c6bfa397b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - typing_extensions
+
+test:
+  imports:
+    - sliceable_dict
+
+about:
+  home: https://github.com/
```

Comment:
This should be changed to an page for the package rather than the entirity of GitHub.

Reply:
```suggestion
  home: https://github.com/palmb/slice-dict
```